articles/api-auth/using-rules/client-credentials.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Describes using rules with Client Credentials Grant.
+---
+
+# Using Rules with Client Credentials Grants
+
+You can now add [rules](/rules) into the [client credentials](/api-auth/grant/client-credentials) exchange pipeline where you exchange a `client_id` and `secret` for an `access_token`.
+
+## Prior to Beginning Your Configuration
+
+<%= include('./_includes/_prerequisites.md') %>
+
+## Creating the Rule
+
+**Note**: You can only create one rule, which will then be executed for **all** clients and APIs.
+
+### 1. Create the Rule For Use with Webtasks
+
+<%= include('./_includes/_create-rule', {
+	  args: 'client, scope, audience, context, cb'
```

Comment:
Resource Owner works fine, my test was wrong. I used the same webtask with client credentials, and in the RO case this is missing the `user` input argument.